examples/DataApi-Voting-poll/scripts/send_data_transaction_vote_poll.js

```diff
@@ -0,0 +1,98 @@
+const { dag4 } = require( '@stardust-collective/dag4' );
+const jsSha256 = require( 'js-sha256' );
+const jsSha512 = require( 'js-sha512' );
+const EC = require( 'elliptic' );
+const axios = require( 'axios' );
+
+const curve = new EC.ec( 'secp256k1' );
+
+/** Encode message according with serializeUpdate on your template module l1 */
+const getEncoded = ( value ) => {
+    const energyValue = JSON.stringify( value );
+    return energyValue;
+};
+
+const serialize = ( msg ) => {
+    const coded = Buffer.from( msg, 'utf8' ).toString( 'hex' );
+    return coded;
+};
+
+const sha256 = ( hash ) => {
+    return jsSha256.sha256( hash );
+};
+
+const sha512 = ( hash ) => {
+    return jsSha512.sha512( hash );
+};
+
+const sign = async ( privateKey, msg ) => {
```

Comment:
Does `dag4.keystore.sign` work here? Either that or we should migrate this to a library somewhere.

Reply:
Done